AssetStudio/AssetsManager.cs

```diff
@@ -231,6 +235,49 @@ private void LoadWebFile(FileReader reader)
             }
         }
 
+        private void LoadZipFile(FileReader reader)
+        {
+            Logger.Info("Loading " + reader.FileName);
+            try
+            {
+                using (ZipArchive archive = new ZipArchive(reader.BaseStream, ZipArchiveMode.Read))
+                {
+                    foreach (ZipArchiveEntry entry in archive.Entries)
+                    {
+                        try
+                        {
+                            string dummyPath = Path.Combine(Path.GetDirectoryName(reader.FullPath), reader.FileName, entry.FullName);
+                            // create a new stream
+                            // - to store the deflated stream in
+                            // - to keep the data for later extraction
+                            Stream streamReader = new MemoryStream();
+                            entry.Open().CopyTo(streamReader);
+                            streamReader.Position = 0;
+
+                            FileReader entryReader = new FileReader(dummyPath, streamReader);
+                            LoadFile(entryReader);
+                        }
+                        catch (Exception e)
+                        {
+                            Logger.Error($"Error while reading zip entry {entry.FullName}", e);
+                        }
+                        finally
+                        {
+                            reader.Dispose();
+                        }
```

Comment:
Can't release the zip reader in the loop.

Reply:
Oh, yeah, indeed.
stupid mistake